_includes/markdown/Markup.md

```diff
@@ -285,10 +285,64 @@ Combining automated and manual testing practices allows 10up to maintain a high
 <h2 id="progressive-enhancement" class="anchor-heading">Progressive Enhancement {% include Util/top %}</h2>
 Progressive enhancement means building a website that is robust, fault tolerant, and accessible. Progressive enhancement begins with a baseline experience and builds out from there, adding features for browsers that support them. It does not require us to select supported browsers or revert to table-based layouts. Baselines for browser and device support are set on a project-by-project basis.
 
-At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible. For example, browser support for SVG has not yet reached 100%. When using SVG you should always provide a fallback such as a PNG image for browsers that do not support vector graphics.
+At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible.
 
 ### Polyfills
 When writing markup that does not have wide browser support, using polyfills can help bring that functionality to those older browsers. Providing support for older browsers is incredibly important to the business objectives of our clients. In an effort to prevent code bloat, we only provide polyfills for features that are functionally critical to a site.
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. Here are a few known benefits of SVG:
+
+* __Scalability__ - They look great on retina displays and at any size, i.e. they're resolution independent.
+* __File Size__ - Small file size and compresses well.
+* __Styling__ - Manipulate fill, stroke, and even animate.
+
+### SVG Sprites
+
+It is optimal to group multiple SVG icons that are likely to be used throughout certain templates or even the entire site (i.e. sprite).
+
+Combine them in to an `svg-defs.svg` file, embed the `svg-defs.svg` within the document, and reference them within the template source with `<use>`. The creation of this icon system should be automated.
```

Comment:
@colorful-tones This is a great start to a much-needed topic in our best practices. I don't have much more to add on this section, just a few suggestions.

* Maybe include a section up top on when it's preferable to use SVGs over PNGs or JPEGs, and vice versa. You do mention "rich vector graphics" and much of the section talks about using SVGs for things like icons, but I have seen code in projects inherited from other agencies or clients that try to make raster images into SVG (and end up embedding a PNG or JPEG inside an SVG file, which I believe is an anti-pattern for most use-cases). Let's try to make it abundantly clear when it is appropriate to use an SVG over the classic formats, vs. cases when a raster image might be the better format.
* Consider adding a link under Further Reading to [MDN's tutorial page on SVGs](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial), or another deep dive into SVG syntax of your choice. This might be useful for helping readers to understand the syntax of SVGs. SVG markup can often be an enigma for newcomers, because it looks almost like HTML but not quite, but it's really useful to be able to read SVG markup and understand what it's doing at a high level.
* I would also love to see a corresponding section on when to use SVGs within CSS instead of embedding into the markup, as well as how embedded SVGs might be styled or animated via CSS. Maybe that's more appropriate for the CSS page. It could be linked to from here near where you mention that you can "Manipulate fill, stroke, and even animate". This could be outside the scope of this PR—let me know if you want me to file an issue on this instead.

Great work!

Reply:
@dkoo 

> Consider adding a link under Further Reading to [MDN's tutorial page on SVGs](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial)...

I added a link to MDN's SVG Tutorial in the "Further Reading" section: https://github.com/10up/Engineering-Best-Practices/pull/280/commits/6a03a6f66b5590d04cadd18a2e3768242f9a4d71

> I have seen code in projects inherited from other agencies or clients that try to make raster images into SVG (and end up embedding a PNG or JPEG inside an SVG file, which I believe is an anti-pattern for most use-cases). Let's try to make it abundantly clear when it is appropriate to use an SVG over the classic formats, vs. cases when a raster image might be the better format.

I have encountered this quite a bit as well. I'm struggling to find a way to add a section though that makes this abundantly clear to be honest. 

Do you think an additional sub-section could be added: "SVG vs other image formats"? Or, something along those lines. Thoughts?

> I would also love to see a corresponding section on when to use SVGs within CSS

In one of the many previous iterations of this section I had something that covered this. 

> ... as well as how embedded SVGs might be styled or animated via CSS. Maybe that's more appropriate for the CSS page.

I believe this was covered in a previous iteration as well. This section has been refocused a few times. It is hard because the section resides on the Markup page. However, one might argue there could be a separate Images page altogether with SVG as a section. 

> It could be linked to from here near where you mention that you can "Manipulate fill, stroke, and even animate". This could be outside the scope of this PR—let me know if you want me to file an issue on this instead.

I agree this is all helpful information and I'm not certain where it should all reside or be organized.

There could likely be an Animations page. 🤔 

Thanks for all this feedback.

Reply:
> I have encountered this quite a bit as well. I'm struggling to find a way to add a section though that makes this abundantly clear to be honest.
> 
> Do you think an additional sub-section could be added: "SVG vs other image formats"? Or, something along those lines. Thoughts?

Yeah, I think a new sub-section right under the known benefits list would make sense to me. I like your suggested sub-section title.

As for the content on CSS and animations, I can see what you mean about the page being refocused multiple times. Let's table that for inclusion in another section, maybe under the CSS page. I'll create an issue so we don't forget to discuss.

Reply:
@dkoo I added a small section on "potential drawbacks", as opposed to the original idea of separate "SVG vs other image formats". 

Let me know what you think? https://www.dropbox.com/s/sxy6u1sadyu7i2l/Screenshot%202019-02-22%2016.21.43.png?dl=0

Also, thanks for creating the new, separate #289 👍